packages/plugin-trpc/web/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "webstone-plugin-web-trpc",
+	"version": "0.0.1",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "svelte-kit sync && svelte-package",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
```

Comment:
I removed the `test` script here, also because of the issue we've had before with the pre-commit
The command executed is `playwright test`